ISISComputingGroup/ibex_gui
# Ticket1350 delete multiple synoptics

###Description of work

ISISComputingGroup/IBEX/issues/1350

###To test

1. Attempt to delete multiple synoptics and configurations.
2. Ensure that multiple configs or synoptics cannot be edited.
3. Ensure that the system tests work as intended and pass. (https://github.com/ISISComputingGroup/ibex_system_tests/pull/20)

###Acceptance criteria

1. Multiple synoptics can now be deleted at once (by holding ctrl and/or shift click)
2. Multiple configurations can now be deleted at once (by holding ctrl and/or shift click)

#### Code Review


### Functional Tests
